Content.Client/Chat/ChatBox.cs

```diff
@@ -59,6 +73,39 @@ protected override void Initialize()
             Input.OnTextEntered += Input_OnTextEntered;
             vBox.AddChild(Input);
 
+            vBox.AddChild(hBox);
+
+            AllButton = new Button()
+            {
+                Text = "ALL",
```

Comment:
You should localize this string and the one below it with `ILocalizationManager.GetString()`.